tests/std/tests/P2474R2_views_repeat/test.cpp

```diff
@@ -360,6 +360,16 @@ constexpr bool test() {
     test_iterator_arithmetic<wchar_t>();
     test_iterator_arithmetic<_Signed128>();
 
+    // GH-4507: LWG-4053 Unary call to std::views::repeat does not decay the argument
+    {
+        using RPV = ranges::repeat_view<const char*>;
+
+        static_assert(same_as<decltype(views::repeat("foo", unreachable_sentinel)), RPV>);

```

Comment:
Oops, these `static_assert`'s actually tested the changes in LWG-4054, not LWG-4053 only. I'll fix this in a new PR soon.